modules/private-clusters-setting-api-private.adoc

```diff
@@ -155,6 +113,48 @@ providerSpec:
 .... Repeat this process for each of the control plane machines.
 endif::post-install[]
 
+. In the web portal or console for your cloud provider, take the following actions:
+
+.. Locate and delete the appropriate load balancer component:
+ifndef::cpmso-using-azure[]
+*** For AWS, delete the external load balancer. The API DNS entry in the private zone already points to the internal load balancer, which uses an identical configuration, so you do not need to modify the internal load balancer.
+endif::cpmso-using-azure[]
+ifndef::cpmso-using-aws[]
```

Comment:
for some reason these are not showing up as correctly highlighted in the diff and the vale bot seems to be ignoring these when checking for balanced ifdefs. That said, the rendered preview works as intended with these statements and the vale errors can be ignored.